.github/workflows/weekly_updates.yml

```diff
@@ -26,7 +26,8 @@ jobs:
 
       - name: Install dependencies
         working-directory: ./automations/js
-        run: npm ci
+        shell: bash
+        run: "pnpm install"
```

Comment:
Why does `shell` need to be specified here? Can't we just run `pnpm install`?

Reply:
I'm honestly not sure. I recall there may have been a problem without it, as this shell configuration is present in our frontend repo action that behaves similarly to this one.

Reply:
The `shell: bash` line is [optional](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell).

It's best to be clear if we are using some bash specific syntax but we aren't here.